test/infrastructure/rbac.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/component: dashboard
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-dashboard
+  name: tekton-pipelines-executor
```

Comment:
This is pretty clear to me that this is used for our execution tasks. Pipelines are used to organize these tasks. I'm not sure a comment would be too useful here especially since it's in the rbac.yaml file as well. What info are you hoping to clear up?